bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/cfi/ContentFragmentImport.java

```diff
@@ -90,9 +90,10 @@
 
     @FormField(
             name = "Dry run",
-            component = CheckboxComponent.class
+            component = CheckboxComponent.class,
+            options = "checked"
     )
-    transient boolean dryRunMode;
+    transient boolean dryRunMode = true;
```

Comment:
Why is this instance transient?

Reply:
Because it was already that way, I was just setting the bool to true and didn't know if the original author had it transient for a reason.